.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
+.buildinfo
+.git
+.nojekyll
+objects.inv
+.vscode/
```

Comment:
The project gitignore should only contain ignores relevant to the project itself so you shouldn't need to add the following:

- `.vscode/`
- `.DS_Store`

These should be ignored in your global gitignore configuration.

Reply:
Ok, sorry. Updated the .gitignore accordingly.

Reply:
`.git` should not need to go into the `.gitignore` either. Git knows to ignore itself.

What are the other things in this listing you added?